lib/CMakeLists.txt

```diff
@@ -1,31 +1,12 @@
-# cmake control file for libdattobd
-set(CMAKE_VERBOSE_MAKEFILE ON)
-
-cmake_minimum_required(VERSION 2.8)
-
-SET(CMAKE_BUILD_TYPE Release)
-
-IF(DEFINED CMAKE_BUILD_TYPE)
-   SET(CMAKE_BUILD_TYPE ${CMAKE_BUILD_TYPE} CACHE STRING "Choose the type of build, options are: Debug Release RelWithDebInfo")
-   MESSAGE ("Using existing build type.")
-ELSE()
-   SET(CMAKE_BUILD_TYPE RelWithDebInfo CACHE STRING "Choose the type of build, options are: Debug Release RelWithDebInfo")
-   MESSAGE ("Setting build type.")
-ENDIF()
-
-MESSAGE ("Build type: ${CMAKE_BUILD_TYPE}")
+cmake_minimum_required(VERSION 2.8.11)
 
 project(dattobd)
```

Comment:
This needs to be set to `project(dattobd C)` so that CMake won't require a C++ compiler for this project.